app/src/main/java/com/sebastianvm/mango/database/MangoDatabase.kt

```diff
@@ -12,7 +12,7 @@ import dagger.hilt.InstallIn
 import dagger.hilt.android.qualifiers.ApplicationContext
 import dagger.hilt.components.SingletonComponent
 
-@Database(entities = [JobEntity::class], version = 1)
+@Database(entities = [JobEntity::class], version = 1, exportSchema = false)
```

Comment:
Since the db schema will go through many changes soon, let's keep this as false for now. Once we are somewhat stable, we an begin exporting the schema.